lib/galaxy/tools/__init__.py

```diff
@@ -234,7 +234,6 @@
     "send_to_cloud",
     "__DATA_FETCH__",
     "directory_uri",
-    "export_remote",
```

Comment:
Can you please also add it to the `GALAXY_LIB_TOOLS_VERSIONED` dictionary below?

Reply:
done, nice catch 🦅 👁️